run_sorting_hat_test_data.sh

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env bash
 
+NEXPS=${NEXPS:-30}
+NJOBS=${NJOBS:-10}
 GTF=$( pwd )/test-data/gff
-FORCEALL=$true
+FORCEALL=true
```

Comment:
```suggestion
FORCEALL=${FORCEALL:-false}
```
I would leave the least dangerous case as default.

Reply:
On second thoughts, this is no part of the PR, it was probably resolved elsewhere.... I'll remove the commit suggestion. Feel free to merge as it is.